api.bs

```diff
@@ -855,6 +864,52 @@ if the user has opted out of collection of diagnostic data.
 * User ability to view the impression store and past report submissions.
 
 
+# HTTP API # {#http-api}
+
+\`<dfn http-header><code>Save-Impression</code></dfn>\` is a
+[=structured header/dictionary|Dictionary Structured Header=]
+set on a response requesting that the user agent invoke the
+<a method for=PrivateAttribution>saveImpression()</a> API.
+
+<pre class=example id=ex-save-impression-header>
+Save-Impression: conversion-site=advertiser.example, histogram-index=2, \
+  filter-data=12, lifetime-days=7
+</pre>
+
+The following keys are defined, corresponding to the members of
+the {{PrivateAttributionImpressionOptions}} dictionary passed to
+<a method for=PrivateAttribution>saveImpression()</a>.
+
+<dl dfn-for=save-impression>
+  <dt><code>conversion-site</code></dt>
+  <dd>
+    Value of <a dict-member for=PrivateAttributionImpressionOptions>conversionSite</a>,
```

Comment:
For each of these fields, we should clarify what structured header types they permit.

Based on the examples, I see this as:

- `conversion-site` is a [token](https://httpwg.org/specs/rfc9651.html#token)
- `histogram-index` is a non-negative [integer](https://httpwg.org/specs/rfc9651.html#integer)
- `filter-data` is a non-negative integer (per discussion in https://github.com/w3c/ppa/pull/56/files#r1905845657)
- `lifetime-days` is a positive integer

Reply:
It's possible that we will want `conversion-site` to be a string for generality, given that tokens do have some restrictions on their contents.